internal/service/ipam/model_ipamsvc_server.go

```diff
@@ -342,6 +346,15 @@ var IpamsvcServerResourceSchemaAttributes = map[string]schema.Attribute{
 		Required:            true,
 		MarkdownDescription: "The name of the DHCP Config Profile. Must contain 1 to 256 characters. Can include UTF-8.",
 	},
+	"profile_type": schema.StringAttribute{
+		Optional: true,
+		Computed: true,
+		Default:  stringdefault.StaticString("server"),
+		PlanModifiers: []planmodifier.String{
+			stringplanmodifier.RequiresReplaceIfConfigured(),
```

Comment:
is this because we are not allowed to change the `profile_type` once created?

Reply:
Yes , this is implemented as Profile Type is not allowed to be updated . 
Following Error is displayed :
`ProfileType field is not allowed to be updated`